src/main/java/uk/gov/hmcts/reform/professionalapi/service/impl/FeatureToggleServiceImpl.java

```diff
@@ -40,13 +40,14 @@ public void mapServiceToFlag() {
         launchDarklyMap = new HashMap<>();
         launchDarklyMap.put("OrganisationMfaStatusController.retrieveMfaStatusByUserId", "prd-mfa-flag");
         launchDarklyMap.put("OrganisationInternalController.updateOrgMfaStatus", "prd-mfa-flag");
+        launchDarklyMap.put("OrganisationExternalController.deletePaymentAccountsOfOrganisation",
+                "rd-professional-multi-pba");
     }
 
     @Override
     public boolean isFlagEnabled(String serviceName, String flagName) {
         LDUser user = new LDUser.Builder(userName)
             .firstName(userName)
-            .custom("servicename", serviceName)
```

Comment:
is this no longer required?

Reply:
It is not required as `ExUI` is the only consumer of this API. @shreedhar-hmcts Please confirm